### PR TITLE
wbaas-backup: Reduce cpu and memory limits

### PR DESCRIPTION
**Pull request by deer-wmde**
Reduces cpu and memory limits in wbaas-backup chart

### CODE REVIEW

File: charts/wbaas-backup/templates/_helpers.tpl [hunk 1]
```diff
@@ -28,8 +28,8 @@ lifecycle:
 {{- end }}
 resources:
   limits:
-    cpu: 750m
```
Comment (toban) [inline charts/wbaas-backup/templates/_helpers.tpl:31]: I'm pretty sure the CPU is well over this new limit, did you try this out somewhere?

Comment (toban) [reply charts/wbaas-backup/templates/_helpers.tpl:31]: image

from staging

Comment (deer-wmde) [reply charts/wbaas-backup/templates/_helpers.tpl:31]: I just looked at these graphs and saw that the `used` value was way lower than the request most of the time, but your screenshot tells another story. Maybe I was looking at the wrong graph?
image

**Comment by rosalieper** [comment]
should this PR be closed?

**Comment by toban** [comment]
> should this PR be closed?

I think it could be probably

**Pull request closed without merging**